recipes/ocl/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "opencamlib" %}
+{% set version = "2018.08" %}
+{% set commit = "8e8337f61b05" %}
+{% set sha256 = "a24cc1e993fe78140e82a8fc8be5bec3edcd9ca4389bd13e61d381d88f8c2c90" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    # fn: {{ name }}.tar.gz
+    # url: https://github.com/aewallin/{{ name }}/archive/{{ version }}.tar.gz
+    # sha256: {{ sha256 }}
+    git_url: https://github.com/aewallin/opencamlib
+    git_tag: master
+
+build:
+    number: 0
+    skip: true  # [py<35]
+
+requirements:
+    build:
+        - {{ compiler("cxx") }}
+        - ninja
+        - cmake
+    host:
+        - python
+        - boost 1.68
+        - doxygen
+    run:
+        - boost 1.68
+        - python
+
+test:
+  imports:
+    - ocl
+
+about:
+    home: http://www.anderswallin.net/CAM/
+    license: GPL3.0
```

Comment:
Could you please change it to: ``GPL-3.0`` ?

Reply:
the license changed to  LGPL-2.1.
https://github.com/aewallin/opencamlib/blob/master/COPYING